solutions/silver/cf-51C.mdx

```diff
@@ -88,6 +92,53 @@ int main() {
 ```
 
 </CPPSection>
+
+<PySection>
+
+```py
+EPSILON = 1e-7
+
+
+def test(d: int, x: list) -> int:
+	used = 0
+	pos = [x[0] + d] * 3
+	"""
+    Checks if each house is within range of the most recently placed
+    station, if not, builds a new one (unless there are none left)
+    """
```

Comment:
weird formatting

Reply:
pre-commit bot error.

Reply:
I tried to add extra spacing to it but still don't know why it is showing like this here

Reply:
given that it's only two lines, you could probably just use two individual comments (assuming precommit is still tweaking out)

Reply:
no python doc comments are just like that
you need a string

Reply:
> I tried to add extra spacing to it but still don't know why it is showing like this here

check ur tabs & spaces

Reply:
corrected it directly from GitHub only.